src/components/custom-constellation/field/HMRC_ODX_GDSTextPresentation/index.jsx

```diff
@@ -0,0 +1,255 @@
+  );
+
+  const [inputValue, setInputValue] = useState(value);
+  const [status, setStatus] = useState(hasSuggestions ? 'pending' : undefined);
+  useEffect(() => setInputValue(value), [value]);
+
+  useEffect(() => {
+    if (validatemessage !== '') {
+      setStatus('error');
+    }
+    if (hasSuggestions) {
+      setStatus('pending');
+    } else if (!hasSuggestions && status !== 'success') {
+      setStatus(validatemessage !== '' ? 'error' : undefined);
+    }
+  }, [validatemessage, hasSuggestions, status]);
+
+  const onResolveSuggestionHandler = accepted => {
+    suggestionsHandler(accepted, pConn, setStatus);
+  };
+
+  // Override the value to render as status work when prop passed to display as status
+  if (displayAsStatus) {
+    value = StatusWorkRenderer({ value });
+
+    // Fall into this scenario for case summary, default to stacked status
+    if (!displayMode) {
+      return (
+        <FieldValueList
+          variant='stacked'
+          data-testid={testId}
+          fields={[{ id: 'status', name: label, value }]}
+        />
+      );
+    }
+  }
+
+  if (displayMode === 'LABELS_LEFT' || displayMode === 'DISPLAY_ONLY') {
+    let displayComp = value || <span aria-hidden='true'>&ndash;&ndash;</span>;
+    if (isTableFormatter && formatExists(formatter)) {
+      displayComp = textFormatter(formatter, value);
+    }
+    return displayMode === 'DISPLAY_ONLY' ? (
+      <StyledHmrcOdxGdsTextPresentationWrapper>
+        {displayComp}
+      </StyledHmrcOdxGdsTextPresentationWrapper>
+    ) : (
+      <StyledHmrcOdxGdsTextPresentationWrapper>
+        <FieldValueList
+          variant={hideLabel ? 'stacked' : variant}
+          data-testid={testId}
+          fields={[{ id: '1', name: hideLabel ? '' : label, value: displayComp }]}
+        />
+      </StyledHmrcOdxGdsTextPresentationWrapper>
+    );
+  }
+
+  if (displayMode === 'STACKED_LARGE_VAL') {
+    const isValDefined = typeof value !== 'undefined' && value !== '';
+    const val = isValDefined ? (
+      <Text variant='h1' as='span'>
+        {value}
+      </Text>
+    ) : (
+      ''
+    );
+    return (
+      <StyledHmrcOdxGdsTextPresentationWrapper>
+        <FieldValueList
+          variant='stacked'
+          data-testid={testId}
+          fields={[{ id: '2', name: hideLabel ? '' : label, value: val }]}
+        />
+      </StyledHmrcOdxGdsTextPresentationWrapper>
+    );
+  }
+
+  return (
+    <StyledHmrcOdxGdsTextPresentationWrapper>
+      <Input
+        {...additionalProps}
+        type='text'
+        label={label}
+        labelHidden={hideLabel}
+        info={validatemessage || helperText}
+        data-testid={testId}
+        value={inputValue}
+        status={status}
+        placeholder={placeholder}
+        disabled={disabled}
+        readOnly={readOnly}
+        required={required}
+        maxLength={maxLength}
+        onChange={event => {
+          if (hasSuggestions) {
+            setStatus(undefined);
+          }
```

Comment:
During initializing and line number 95, you are passing `pending `to `setStatus` when `hasSuggestions` is true.